.travis.yml

```diff
@@ -6,7 +6,8 @@ jdk:
 android:
   components:
     - tools
-    - build-tools-25.0.2
+    - build-tools-27.0.3
+    - build-tools-26.1.3
```

Comment:
build tools doesn't need to be specified anymore, the latest gradle will figure out what to use

Reply:
it will, but it will fail to install it if license is not accepted beforehand, so you either add pre-accepted license or use travis to install it. I preferred travis as it seems simpler.